Zend/zend_enum.c

```diff
@@ -307,10 +307,10 @@ ZEND_API zend_result zend_enum_get_case_by_value(zend_object **result, zend_clas
 		}
 
 		if (ce->enum_backing_type == IS_LONG) {
-			zend_value_error(ZEND_LONG_FMT " is not a valid backing value for enum \"%s\"", long_key, ZSTR_VAL(ce->name));
+			zend_value_error("\"" ZEND_LONG_FMT "\" is not a valid backing value for enum %s", long_key, ZSTR_VAL(ce->name));
```

Comment:
IMO it's better to keep integers unquoted to not confuse them with strings. But let's see what others think.

Reply:
> IMO it's better to keep integers unquoted to not confuse them with strings. But let's see what others think.

I was very much on the fence about this @iluuu1994. On one hand, it's an integer, but on the other hand, it makes it match the string error message. Totally agree, let's see what others think.

Reply:
From a quick look, it seems that we never quote integers in error messages.

Reply:
Will update